lib/screens/alerts/alert.ex

```diff
@@ -421,7 +421,7 @@ defmodule Screens.Alerts.Alert do
     |> Enum.uniq_by(& &1.stop)
   end
 
-  @spec station_closure_type(__MODULE__.t(), list(Stop.t())) ::
+  @spec station_closure_type(__MODULE__.t(), list(Stop.t() | String.t())) ::
```

Comment:
Does this new definition mean we will have a mix of Stops and Strings in the list? 

Also maybe a silly question, is there a difference between list() vs []? Can they be used interchangeably?

Reply:
good catch! this updated spec was from an earlier revision of the code